crowd.properties

```diff
@@ -2,13 +2,20 @@ category=Integration
 description=Provide the ability to authenticate again Crowd
 homepageUrl=https://github.com/deepy/sonar-crowd
 devVersion=
-publicVersions=2.1.2
+publicVersions=2.1.2,2.1.3
 archivedVersions=1.0,2.0,2.1,2.1.1
 privateVersions=
 
 defaults.mavenGroupId=cm.xd.sonar-plugins
 defaults.mavenArtifactId=sonar-crowd-plugin
 
+2.1.3.description=Support SonarQube 6.7-7.7
+2.1.3.sqVersions=[7.9,LATEST]
+2.1.3.date=2019-06-03
```

Comment:
🤔 This date was a month ago!

Reply:
I might've been thinking a bit too far ahead :flushed: